urwid/constants.py

```diff
@@ -0,0 +1,30 @@
+# define some names for these constants to avoid misspellings in the source
+# and to document the constant strings we are using
+
+# Widget sizing methods
+FLOW = 'flow'
+BOX = 'box'
+FIXED = 'fixed'
+
+# Text alignment modes
+LEFT = 'left'
+RIGHT = 'right'
+CENTER = 'center'
+
+# Filler alignment
+TOP = 'top'
+MIDDLE = 'middle'
+BOTTOM = 'bottom'
+
+# Text wrapping modes
+SPACE = 'space'
+ANY = 'any'
+CLIP = 'clip'
+ELLIPSIS = 'ellipsis'
+
+# Width and Height settings
+PACK = 'pack'
+GIVEN = 'given'
+RELATIVE = 'relative'
+RELATIVE_100 = (RELATIVE, 100)
+WEIGHT = 'weight'
```

Comment:
I think these all need to be made into [`Enum`-s](https://docs.python.org/3.4/library/enum.html).

That'd also allow to actually document the constants — via proper docstrings appearing in Sphinx docs, rather than in-source comments.